### ISSUE TITLE
Apply tag color to foreground instead of background? 

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/profileview#55]
The tag color is almost not perceptible in the Tags as currently implemented. (see the leftmost Tag)  Could the color be represented on the tags differently?  

It seems unlikely that an alpha channel blending is workable. (2nd tag) Even if technically doable, the grayscale of the alpha channel overlay changes the color too much.

But maybe an inset rectangle on the body of the white tag would be visible enough. (3rd & 4th tags)  

![image](https://user-images.githubusercontent.com/69127217/126096829-d6f5d81c-cf31-4d09-a739-7a8be923fa00.png)

**Comment by cdhorn** [comment]
Yeah, when the time comes this may get changed. I read somewhere that svg icon colors can be changed so that was a thought I had when first looking at it.
